strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -306,7 +306,7 @@ def prepare_cat(self, alpha, phi, cutoff, desc, D):
         # case alpha = 0 -> prepare vacuum
         if np.isclose(np.absolute(alpha), 0):
             return (
-                np.array([[1]], dtype=complex),
+                np.array([1], dtype=complex),
```

Comment:
eventually we might want to change `complex` to be `np.complex128` or whatever the highest numerical precision available is.